src/wallet/wallet.cpp

```diff
@@ -515,12 +515,29 @@ void CWallet::SetBestChain(const CBlockLocator& loc)
 std::set<std::pair<libzcash::PaymentAddress, uint256>> CWallet::GetNullifiersForAddresses(const std::set<libzcash::PaymentAddress> & addresses)
 {
     std::set<std::pair<libzcash::PaymentAddress, uint256>> nullifierSet;
+    // Sapling ivk -> addr map
+    std::map<libzcash::SaplingIncomingViewingKey, libzcash::SaplingPaymentAddress> ivkMap;
```

Comment:
The below is not correct but will be fixed when https://github.com/zcash/zcash/pull/3499 is merged.